lib/glimesh/accounts/user.ex

```diff
@@ -203,6 +208,31 @@ defmodule Glimesh.Accounts.User do
     |> cast_attachments(attrs, [:avatar])
   end
 
+  def big_scary_changeset(user, attrs) do
```

Comment:
This is not a very descriptive name

Reply:
It used to be more descriptive. Clone wanted it named in a way that made it so it couldn't be used accidentally. Used to be called gct_user_changeset iirc.

Reply:
Looks to be user_changeset before swapping. https://github.com/Glimesh/glimesh.tv/pull/225/commits/bf012c90c501ca8c27ab4270e28964ab41493ff6

Reply:
Can we name it in a way where it's clear what the intended use is? The other names clearly indicate their use - `stripe_changeset`, `profile_changeset`, `password_changeset`, etc.

Reply:
`gct_user_changeset` seems reasonable to me but cc @clone1018